catalyst-gateway/bin/src/service/docs/mod.rs

```diff
@@ -5,27 +5,25 @@ use poem_openapi::{OpenApi, OpenApiService, Webhook};
 use rust_embed::RustEmbed;
 
 /// Create the documentation pages where the `OpenAPI` docs can be viewed.
-pub(crate) fn docs<T: OpenApi + 'static, W: Webhook + 'static>(
-    api_service: &OpenApiService<T, W>,
-) -> Route {
-    let spec = api_service.spec();
-
+pub(crate) fn docs<T, W>(api_service: &OpenApiService<T, W>) -> Route
+where
+    T: OpenApi + 'static,
+    W: Webhook + 'static,
+{
     let swagger_ui = api_service.swagger_ui();
     let rapidoc_ui = api_service.rapidoc();
     let redoc_ui = api_service.redoc();
     let openapi_explorer = api_service.openapi_explorer();
-    let stoplight_ui = stoplight_elements::create_endpoint(&spec);
+    let stoplight_ui = stoplight_elements::create_endpoint(&api_service.spec());
 
     Route::new()
         .at("/", get(stoplight_ui))
         .nest("/swagger_ui", swagger_ui)
         .nest("/redoc", redoc_ui)
         .nest("/rapidoc", rapidoc_ui)
         .nest("/openapi_explorer", openapi_explorer)
-        .at(
-            "/cat-gateway.json",
-            poem::endpoint::make_sync(move |_| spec.clone()),
-        )
+        .at("/cat-gateway.json", api_service.spec_endpoint())
+        .at("/cat-gateway.yml", api_service.spec_endpoint_yaml())
```

Comment:
We do not want two formats for the same data.  The only format should be pretty printed json.